ReactWindows/ReactNative/Common/AccessibilityHelper.cs

```diff
@@ -0,0 +1,255 @@
+        /// <param name="uiElement"></param>
+        public static void InitImportantForAccessibility(UIElement uiElement)
+        {
+            var uiElementAutomationPeer = FrameworkElementAutomationPeer.FromElement(uiElement);
+            if (IsHiddenByAncestor(uiElementAutomationPeer))
+            {
+                // If an ancestor is "hiding" the element, set AccessibilityView to Raw.
+                AutomationProperties.SetAccessibilityView(uiElement, AccessibilityView.Raw);
+                SetChildrenAccessibilityView(uiElementAutomationPeer, AccessibilityView.Raw);
+            }
+        }
+
+        /// <summary>
+        /// Sets the ImportantForAccessibility property for <paramref name="uiElement"/>.
+        /// It uses AutomationProperties.AccessibilityView to implement the element and its children
+        /// visibility to narrator. ImportantForAccessibility value is stored as an attached property.
+        /// </summary>
+        /// <param name="uiElement"></param>
+        /// <param name="importantForAccessibilityNew"></param>
+        public static void SetImportantForAccessibility(UIElement uiElement, ImportantForAccessibility importantForAccessibilityNew)
+        {
+            var importantForAccessibilityOld = GetImportantForAccessibilityAttached(uiElement);
+            if (importantForAccessibilityOld == importantForAccessibilityNew)
+            {
+                return;
+            }
+
+            SetImportantForAccessibilityAttached(uiElement, importantForAccessibilityNew);
+
+            // Check if the element is "hidden" (i.e. AccessibilityView = 'Raw') by an ancestor. This occurs if
+            // there is an ancestor with 'Yes' or 'NoHideDescendants'. In this case make no change to AccessibilityView.
+            // If later the element or its parent ImportantForAccessibility is set to 'Auto' or 'No', that makes children
+            // visible, SetChildrenAccessibilityViewFromImportantForAccessibility will update AccessibilityView properly
+            // for the element and its children accordingly.
+            var uiElementAutomationPeer = FrameworkElementAutomationPeer.FromElement(uiElement);
+            if (IsHiddenByAncestor(uiElementAutomationPeer))
+            {
+                return;
+            }
+
+            // Update AccessibilityView property accordingly.
+            UpdateAccessibilityViewForUIElement(uiElement, uiElementAutomationPeer, importantForAccessibilityNew);
+        }
+
+        /// <summary>
+        /// Attached property used to store ImportantForAccessibility value in native controls.
+        /// </summary>
+        public static readonly DependencyProperty ImportantForAccessibilityAttachedProperty =
+            DependencyProperty.RegisterAttached(
+            "ImportantForAccessibilityAttached",
+            typeof(ImportantForAccessibility),
+            typeof(UIElement),
+            new PropertyMetadata(ImportantForAccessibility.Auto));
```

Comment:
This doesn't appear to be indented properly. The function call and its argument list are indented at the same level.

Reply:
Fixed.

---
In reply to: [164510446](https://github.com/Microsoft/react-native-windows/pull/1595#discussion_r164510446) [](ancestors = 164510446)